sklearn/tree/tests/test_export.py

```diff
@@ -207,6 +208,20 @@ def test_graphviz_toy():
 
     assert_equal(contents1, contents2)
 
+    # Test classifier with degraded learning set
+    clf = DecisionTreeClassifier(max_depth=3)
+    clf.fit(X, y_degraded)
+
+    out = StringIO()
+    export_graphviz(clf, out_file=out, filled=True)
+    contents1 = out.getvalue()
+    contents2 = 'digraph Tree {\n' \
```

Comment:
I've used this line separator because of consistency. All other lines in this file divided with backslash. I think in this case I should replace all backslashes.